src/main/java/controllers/BookstoreController.java

```diff
@@ -133,18 +126,11 @@ public Bookstore createBookstore(@RequestBody Bookstore bookstore){
         return b.getSales();
     }
 
-
-    /**
-     * Deletes a bookstore given the JSON representation as a request body
-     * @param bookstore Serialized bookstore from JSON
-     * @return bookstore if present or null otherwise 
-     */
     @DeleteMapping()
```

Comment:
deleted comments?

Reply:
not sure why that was removed, but it has been fixed